mbed_devices/_internal/resolve_target.py

```diff
@@ -42,19 +47,29 @@ def resolve_target(candidate: CandidateDevice) -> MbedTarget:
         try:
             return get_target_by_product_code(product_code)
         except UnknownTarget:
+            logger.error(f"Could not identify an Mbed Target with the Product Code: '{product_code}'.")
```

Comment:
Is it possible for this error to appear in relation to USB MSD's connected which aren't mbed enabled devices (i.e a memory stick)? Or is there some other filtering of candidates further down the stack which prevents that happening?

Reply:
I ion't think as this information is extracted from the HTM file, USB MSB that have a serial number is later code and that just generates an info message and has different wording.